internal/cli/atlas/backup/exports/buckets/create_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package buckets
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/mocks"
+	"github.com/mongodb/mongodb-atlas-cli/internal/test"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestCreateOpts_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockExportBucketsCreator(ctrl)
+	defer ctrl.Finish()
+
+	expected := &mongodbatlas.CloudProviderSnapshotExportBucket{}
+
+	t.Run("flags run", func(t *testing.T) {
```

Comment:
[q] do you plan an example that doesn't use flags? if no please remove the extra nesting here